### ISSUE TITLE
Training on the provided dataset

### ISSUE THREAD

**Issue by YongchieM** [zju3dv/rnin-vio#4]
Dear author, thank you very much for your work. 
When I trained the model for 70 epochs with the dataset you provided, I found the performance is really poor. I tried to modify the parameter start_cov_epochs = 70 and use NLL loss to continue the training. The val loss also didn't decrease. Do you have some tricks to train the model?

Best

**Comment by danpeng2** [comment]
> Dear author, thank you very much for your work. When I trained the model for 70 epochs with the dataset you provided, I found the performance is really poor. I tried to modify the parameter start_cov_epochs = 70 and use NLL loss to continue the training. The val loss also didn't decrease. Do you have some tricks to train the model?
> 
> Best

**Comment by danpeng2** [comment]
Hello, we don't need some tricks to train the model. You can refer to this issue(https://github.com/zju3dv/rnin-vio/issues/2), it may help you.
